WORKSPACE

```diff
@@ -149,5 +148,24 @@ load("@io_bazel_rules_scala//scala:toolchains.bzl", "scala_register_toolchains")
 scala_register_toolchains()
 
 load("@io_bazel_rules_scala//testing:scalatest.bzl", "scalatest_repositories", "scalatest_toolchain")
+
 scalatest_repositories()
+
 scalatest_toolchain()
+
+RULES_GO_VERSION = "v0.35.0"
```

Comment:
IMO, it would be better to inline this below so the `http_archive` declaration below can be copy+pasted out of the `rules_go` README or release notes. At minimum, we're going to have to copy+paste the SHA-256, so why not the whole thing?